libs/live-signer-evm/src/LegacySignerEth.ts

```diff
@@ -0,0 +1,54 @@
+import { EvmAddress, EvmSignature, EvmSigner } from "@ledgerhq/coin-evm/lib/types/signer";
+import Eth from "@ledgerhq/hw-app-eth";
+import { EIP712Message } from "@ledgerhq/types-live";
+import { ResolutionConfig } from "@ledgerhq/hw-app-eth/lib/services/types";
+
+export class LegacySignerEth implements EvmSigner {
```

Comment:
[ASK] Same remark between `EvmSigner` and `Eth` knowing there is no logic in this classe except just calling methods of the `Eth` object

Reply:
Was not sure about this one as Eth and EvmSigner are compatible, maybe it's better to just return `Eth` class as `LegacySignerEth` but here we ensure that `LegacySignerEth` implement the same interface as `DmkSignerEth`.

Reply:
Also, `Eth` cannot implements `SignerEvm` as it's not possible to import `coin-evm` from `hw-app-eth` (circular dependencies)

Reply:
Ok I think I got it. So let's keep things like this 😉